include/picongpu/unitless/simulation.unitless

```diff
@@ -127,6 +135,50 @@ namespace picongpu
             {
                 return static_cast<T_Type>(si.getClassicalElectronRadius() / unit.length());
             }
+
+            struct Convert
+            {
+                /** convert int the pic unit system
+                 *
+                 * @{
+                 */
+                template<typename T_Type = float_X>
+                constexpr T_Type eV2Joule(T_Type eV = 1.0) const
```

Comment:
```suggestion
                constexpr T_Type eV2picEnergy(T_Type eV = 1.0) const
```

Reply:
but be careful when using automated search and replace since eV2Joule also exist in the SI namespace.

Reply:
also needs to be renamed in the other functions

Reply:
offline discussed. Other functions are named equally. I am open that someone refactor it in a sperate PR because this is out of the scope of this PR.